src/main/java/edu/greenblitz/tobyDetermined/subsystems/telescopicArm/Elbow.java

```diff
@@ -0,0 +1,89 @@
+package edu.greenblitz.tobyDetermined.subsystems.telescopicArm;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.tobyDetermined.subsystems.GBSubsystem;
+import edu.greenblitz.utils.motors.GBSparkMax;
+
+
+public class Elbow extends GBSubsystem {
+
+
+    private static Elbow instance;
+    public ElbowState state = ElbowState.IN_ROBOT;
+    public GBSparkMax motor;
+
+
+
+    public static Elbow getInstance(){
+        if(instance == null){
+            instance = new Elbow();
+        }
+        return instance;
+    }
+
+    private Elbow(){
+        motor = new GBSparkMax(RobotMap.telescopicArm.elbow.MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+        motor.config(new GBSparkMax.SparkMaxConfObject()
```

Comment:
add general configuration stuff take insperation from the chassis configs

Reply:
what?